README.md

```diff
@@ -1,2 +1,5 @@
 # Juntos
 Juntos is a web application that lets you co watch streaming service such as youtube. More services to be added in future such as Netflix, Prime, HBO NOW ...etc.
+
+# Artefacts
+## <a href="prd/Juntos System Design.pdf">System Design</a>
```

Comment:
> Please verify the changes.
The link isn't valid.

Reply:
resolved the link issue, changed file name and used markdown link syntax